css/active.css

```diff
@@ -0,0 +1,74 @@
+.back-icon {
```

Comment:
remove css files from pull-request because you use a scss and gulp builds all styles files

Reply:
done